source/guides/continuous-integration/aws-codebuild.md

```diff
@@ -0,0 +1,248 @@
+    runtime-versions:
+      nodejs: latest
+    commands:
+      - npm ci
+  pre_build:
+    commands:
+      - npm run cy:verify
+      - npm run cy:info
+  build:
+    commands:
+        - npm run start:ci &
+        - npx cypress run --record
+```
+
+{% note success Try it out %}
+To try out the example above yourself, fork the {% url "Cypress Kitchen Sink" https://github.com/cypress-io/cypress-example-kitchensink %} example project and place the above {% url "AWS CodeBuild" https://aws.amazon.com/codebuild/ %} configuration in `buildspec.yml`.
+{% endnote %}
+
+**How this buildspec works:**
+
+- On *push* to this repository, this job will provision and start AWS-hosted Amazon Linux instance with Node.js for running the outlined `pre_build` and `build` for the declared commands within the `commands` section of the configuration.
+- {% url "AWS CodeBuild" https://aws.amazon.com/codebuild/ %} will checkout our code from our GitHub repository.
+- Finally, our `buildspec.yml` configuration will:
+  - Install npm dependencies
+  - Start the project web server (`npm start:ci`)
+  - Run the Cypress tests within our GitHub repository within Electron.
+
+# Testing in Chrome and Firefox with Cypress Docker Images
+
+{% note info %}
+As of version 0.2, CodeBuild does not provide a way to specify a custom image for single build configurations.  One way to solve this is using an {% url "AWS CodeBuild build-list strategy" https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list %}.
+{% endnote %}
+
+AWS CodeBuild offers a {% url "build-list strategy" https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list %} of different job configurations for a single job definition.
+
+The {% url "build-list strategy" https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list %} offers a way to specify an image hosted on DockerHub or the {% url "Amazon Elastic Container Registry (ECR)" https://aws.amazon.com/ecr/ %}.
+
+The Cypress team maintains the official {% url "Docker Images" https://github.com/cypress-io/cypress-docker-images %} for running Cypress locally and in CI, which are built with Google Chrome and Firefox. For example, this allows us to run the tests in Firefox by passing the `--browser firefox` attribute to `cypress run`.
+
+{% note success Cypress Amazon Public ECR %}
+The Cypress team has published it's {% url "Docker Images" https://github.com/cypress-io/cypress-docker-images %} for running Cypress locally and in CI in the {% url "Amazon ECR Public Gallery" https://gallery.ecr.aws %}.
+
+The images are available in the following {% url "Amazon ECR Public Galleries" https://gallery.ecr.aws %}:
+
+- https://gallery.ecr.aws/cypress-io/cypress/base
+- https://gallery.ecr.aws/cypress-io/cypress/browsers
+- https://gallery.ecr.aws/cypress-io/cypress/included
+
+{% endnote %}
+
+
+{% note info %}
```

Comment:
We can label the note blocks: `{% note info "Picking the right Docker Image" %}`. Something like that.